automaxlair/max_lair_instance.py

```diff
@@ -242,11 +244,97 @@ def update_paths(
                 self.paths[2][1].downstream_nodes.append(self.paths[3][2])
             elif self.path_type == '3x2':
                 self.paths[1][0].downstream_nodes.append(self.paths[2][2])
-                self.paths[2][1].downstream_nodes.append(self.paths[3][0])
+                self.paths[2][1].downstream_nodes.append(0, self.paths[3][0])
```

Comment:
Great find. I think `append` needs to be changed to `insert`.

Reply:
indeed :D I just crashed with TypeError: append() takes exactly one argument (2 given)